clang/lib/Sema/SemaCUDA.cpp

```diff
@@ -227,6 +227,10 @@ SemaCUDA::CUDAVariableTarget SemaCUDA::IdentifyTarget(const VarDecl *Var) {
 //
 // Priority order: N, SS, HD, WS, --
 //
+// Note: we deviate from the actual meaning for
+//  N, SS, HD, WS, --.
+// They should be understood as 1st, 2nd or 3rd choice.
```

Comment:
This comment could use some more elaboration. There are five values listed, but only three are ranked, presumably because `WS` and `--` are invalid scenarios. I'm also not sure how to apply the ranking in the table below. I think I understand the intent from discussion elsewhere, but I'm having a hard time seeing it in the table.